DuckDuckGo/OnboardingExperiment/OnboardingIntro/OnboardingView.swift

```diff
@@ -27,37 +27,63 @@ struct OnboardingView: View {
     @ObservedObject private var model: OnboardingIntroViewModel
 
+    @State private var showIntroViewContent = true
+    @State private var showIntroButton = false
+    @State private var animateIntroText = true
+    @State private var showComparisonButton = false
+    @State private var animateComparisonText = false
+
     init(model: OnboardingIntroViewModel) {
         self.model = model
     }
 
     var body: some View {
-        switch model.state {
-        case .landing:
-            backgroundWrapped(view: landingView)
-        case let .onboarding(viewState):
-            backgroundWrapped(view: mainView(state: viewState))
+        ZStack {
+            OnboardingBackground()
+
+            switch model.state {
+            case .landing:
+                landingView
+            case let .onboarding(viewState):
+                onboardingDialogView(state: viewState)
+            }
         }
     }
 
-    private func backgroundWrapped(view: some View) -> some View {
-        view.background(OnboardingBackground())
-    }
-
-    private func mainView(state: ViewState.Intro) -> some View {
+    private func onboardingDialogView(state: ViewState.Intro) -> some View {
         GeometryReader { geometry in
             VStack(alignment: .center) {
-                switch state {
-                case .startOnboardingDialog:
-                    introView
-                        .frame(width: geometry.size.width)
-                case .browsersComparisonDialog:
-                    browsersComparisonView
-                        .frame(width: geometry.size.width)
-                }
+                DaxDialogView(
+                    logoPosition: .top,
+                    onTapGesture: {
+                        withAnimation {
+                            switch model.state {
+                            case .onboarding(.startOnboardingDialog):
+                                showIntroButton = true
+                                animateIntroText = false
+                            case .onboarding(.browsersComparisonDialog):
+                                showComparisonButton = true
+                                animateComparisonText = false
+                            default: break
+                            }
+                        }
+                    },
+                    content: {
+                        VStack {
+                            switch state {
+                            case .startOnboardingDialog:
+                                introView
+                            case .browsersComparisonDialog:
+                                browsersComparisonView
+                            }
+                        }
+                    }
+                )
             }
+            .frame(width: geometry.size.width, alignment: .center)
             .offset(y: geometry.size.height * Metrics.dialogVerticalOffsetPercentage.build(v: verticalSizeClass, h: horizontalSizeClass))
```

Comment:
I used a Spacer initially. The issue is that the design for iPad has the dialogs “almost” centered vertically on screen. For this reason I had to leave the offset.